doc/reference/reference_lua/other.rst

```diff
@@ -140,59 +140,101 @@ and variables which are outside all modules.
 
 .. data:: package.path
 
-    This is a string that Tarantool uses to search for Lua modules,
-    especially important for ``require()``.
-    See :ref:`Modules, rocks and applications <app_server-modules>`.
+    Get directories in which Tarantool searches for Lua :ref:`modules <app_server-modules>`.
+    For example, these directories are used to find modules loaded using ``require()``.
+
+    Suppose the application has the following structure:
+
+    .. code-block:: none
+
+        /home/testuser/myapp
+        ├── .rocks/share/tarantool/
+        │   └── foo.lua
+        ├── init.lua
+        └── modules
+            └── bar.lua
+
+    If you :ref:`run the application <app_server-launching_app_binary>` using the ``tarantool`` command, ...
+
+    .. code-block:: console
+
+        tarantool init.lua
+
+    ... the ``/home/testuser/myapp`` path is added to search paths for modules automatically.
```

Comment:
In this example we're already in the `/home/testuser/myapp` directory (according to the path to `init.lua`), so it would work even before https://github.com/tarantool/tarantool/commit/009d3d9b88786f844b7ef87759396a97461637e7.